Description: Bump soname and symbol-versioning.
 Fix the symbol versioning to use a different version than in gnutls26,
 since this breaks the ABI also bump the soname.
 Use a Debian specific soname and symbol-versioning tag, to make sure 
 that there is no conflict with future upstream versions.
Author: Andreas Metzler <ametzler@debian.org>
Bug-Debian: http://bugs.debian.org/7748742
Origin: vendor
Forwarded: http://lists.gnutls.org/pipermail/gnutls-devel/2014-May/006938.html
Last-Update: 2014-09-18

Index: b/lib/Makefile.am
===================================================================
--- a/lib/Makefile.am	2018-10-22 14:44:03.986028336 -0400
+++ b/lib/Makefile.am	2018-10-22 14:44:03.986028336 -0400
@@ -123,6 +123,7 @@ libgnutls_la_SOURCES = $(HFILES) $(COBJE
 	gnutls.asn pkix.asn libgnutls.map
 
 libgnutls_la_LDFLAGS = -no-undefined \
+		       -release deb0 \
 	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE)
 
 libgnutls_la_LIBADD = ../gl/libgnu.la x509/libgnutls_x509.la \
Index: b/lib/libgnutls.map
===================================================================
--- a/lib/libgnutls.map	2018-10-22 14:44:03.986028336 -0400
+++ b/lib/libgnutls.map	2018-10-22 14:44:36.382538024 -0400
@@ -18,7 +18,7 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>
 
-GNUTLS_1_4
+GNUTLS_DEBIAN_0_1_4
 {
   global:
     gnutls_alert_get;
@@ -501,7 +501,7 @@ GNUTLS_1_4
     *;
 };
 
-GNUTLS_2_8
+GNUTLS_DEBIAN_0_2_8
 {
   global:
     gnutls_certificate_set_x509_simple_pkcs12_mem;
@@ -535,9 +535,9 @@ GNUTLS_2_8
     gnutls_session_ticket_key_generate;
     gnutls_session_ticket_enable_client;
     gnutls_session_ticket_enable_server;
-} GNUTLS_1_4;
+} GNUTLS_DEBIAN_0_1_4;
 
-GNUTLS_2_10
+GNUTLS_DEBIAN_0_2_10
 {
   global:
     gnutls_x509_crt_get_issuer_alt_name2;
@@ -563,9 +563,9 @@ GNUTLS_2_10
     gnutls_hmac_output;
     gnutls_certificate_set_verify_function;
     gnutls_safe_renegotiation_status;
-} GNUTLS_2_8;
+} GNUTLS_DEBIAN_0_2_8;
 
-GNUTLS_2_12
+GNUTLS_DEBIAN_0_2_12
 {
   global:
 	gnutls_certificate_set_retrieve_function;
@@ -653,9 +653,9 @@ GNUTLS_2_12
 	gnutls_certificate_get_issuer;
 	gnutls_x509_crq_verify;
 	gnutls_global_set_time_function;
-} GNUTLS_2_10;
+} GNUTLS_DEBIAN_0_2_10;
 
-GNUTLS_3_0_0 {
+GNUTLS_DEBIAN_0_3_0_0 {
   global:
 	gnutls_x509_trust_list_verify_crt;
 	gnutls_x509_trust_list_add_crls;
@@ -776,9 +776,9 @@ GNUTLS_3_0_0 {
 	gnutls_certificate_set_x509_system_trust;
 	gnutls_session_set_premaster;
 	gnutls_ocsp_resp_check_crt;
-} GNUTLS_2_12;
+} GNUTLS_DEBIAN_0_2_12;
 
-GNUTLS_3_1_0 {
+GNUTLS_DEBIAN_0_3_1_0 {
   global:
         gnutls_pkcs11_get_pin_function;
 	gnutls_pkcs11_obj_list_import_url2;
@@ -1031,9 +1031,9 @@ GNUTLS_3_1_0 {
 	gnutls_pkcs11_copy_x509_privkey2;
 	gnutls_pkcs11_privkey_generate3;
 	gnutls_pkcs11_obj_set_info;
-} GNUTLS_3_0_0;
+} GNUTLS_DEBIAN_0_3_0_0;
 
-GNUTLS_FIPS140 {
+GNUTLS_DEBIAN_0_FIPS140 {
 	gnutls_cipher_self_test;
 	gnutls_pk_self_test;
 	gnutls_mac_self_test;
@@ -1056,7 +1056,7 @@ GNUTLS_FIPS140 {
 	_gnutls_ecdh_compute_key;
 };
 
-GNUTLS_PRIVATE {
+GNUTLS_DEBIAN_0_PRIVATE {
   global:
     # Internal symbols needed by libgnutls-extra:
     _gnutls_log_level;
